src/prepare.js

```diff
@@ -59,15 +57,15 @@ const prepare = async (
     if (oldChartIndexFile) {
         await fs.writeFile(
             path.join(tempDir, "current-index.yaml"),
-            oldChartIndexFile
+            oldChartIndexFile,
         );
 
-        return await helmRepoIndex(tempDir, chartRepoUrlPrefix, "current-index.yaml");
+        return helmRepoIndex(tempDir, chartRepoUrlPrefix, "current-index.yaml");
```

Comment:
Where did you move the `await` handler for the `execa` returned by this function?